keyboards/romac/README.md

```diff
@@ -0,0 +1,16 @@
+
+# RoMac
+
+![RoMac](https://imgur.com/a/W7LUCvS)
+
+A “Plaid” Inspired 12-Key (3x4) Macropad.
+
+Keyboard Maintainer: [Garret G.](https://github.com/garretgartnwr)
+Hardware Supported: RoMac rev.1, rev.2, Pro Micro, Elite-C, Proton C, BlueMicro
+Hardware Availability: Through GB orDirect Message(If Extra Stock is available.
+
+Make example for this keyboard (after setting up your build environment):
+
+    make romac/rev2:default
```

Comment:
You haven't actually set up the directory structure to use different folders for each revision. (See [Clueboard 66%](https://github.com/qmk/qmk_firmware/tree/master/keyboards/clueboard/66) or [KBDfans KBD75](https://github.com/qmk/qmk_firmware/tree/master/keyboards/kbdfans/kbd75) for examples.)

If the two revisions can run identical firmware, you don't need to specify a revision folder or update the directory structure.

```suggestion
    make romac:default
```

If the two revisions *cannot* run identical firmware (firmware for one won't work on the other or vice versa), the directory tree will need to be updated to support both revisions.

Please advise.